kbase-extension/static/kbase/js/common/cellComponents/actionButtons.js

```diff
@@ -28,33 +24,35 @@ define([
          *      - label - button text
 
          */
-        constructor(options) {
-            this.bus = options.bus;
-            this.ui = options.ui;
-            this.runAction = options.runAction;
-            this.actionButtons = options.actions;
-        }
 
-        buildLayout(events) {
+        var t = html.tag,
+            div = t('div');
+
+        const actionButtons = config.actionButtons,
+            ui = config.ui,
+            bus = config.bus,
+            runAction = config.runAction;
+
+        function buildLayout(events) {
             return div({
-                class: `${cssCellType}-action-button__container`,
+                class: 'btn-action__container',
```

Comment:
how about making a variable for the base button name, and then using templates to fill in the various class names? i.e.
```
// somewhere where it'll be in scope for the whole file
const buttonName = 'kb-btn-action';

// example function
function buildLayout(args) {
  return div({ class: `${buttonName}__container` },
    div({ class: `${buttonName}__list` }),
    ...
  );
```

Reply:
I don't think there is a reason to. It will always be the same class.